src/test/java/com/google/jenkins/plugins/computeengine/InstanceConfigurationTest.java

```diff
@@ -184,68 +187,75 @@ public void testConfigRoundtrip() throws Exception {
+            .findFirst();
+    assertTrue(sshKey.isPresent());
+    assertFalse(sshKey.get().isEmpty());
 
     // Network
-    assert (i.getNetworkInterfaces().get(0).getSubnetwork().equals(SUBNETWORK_NAME));
-    assert (i.getNetworkInterfaces()
-        .get(0)
-        .getAccessConfigs()
-        .get(0)
-        .getType()
-        .equals("ONE_TO_ONE_NAT"));
-    assert (i.getNetworkInterfaces()
-        .get(0)
-        .getAccessConfigs()
-        .get(0)
-        .getName()
-        .equals("External NAT"));
+    assertTrue(i.getNetworkInterfaces().get(0).getSubnetwork().equals(SUBNETWORK_NAME));
+    assertTrue(
+        i.getNetworkInterfaces()
+            .get(0)
+            .getAccessConfigs()
+            .get(0)
+            .getType()
+            .equals("ONE_TO_ONE_NAT"));
+    assertTrue(
+        i.getNetworkInterfaces().get(0).getAccessConfigs().get(0).getName().equals("External NAT"));
 
     // Tags
-    assert (i.getTags().getItems().size() == NETWORK_TAGS.split(" ").length);
-    assert (i.getTags().getItems().get(0).equals(NETWORK_TAGS.split(" ")[0]));
-    assert (i.getTags().getItems().get(1).equals(NETWORK_TAGS.split(" ")[1]));
+    assertTrue(i.getTags().getItems().size() == NETWORK_TAGS.split(" ").length);
+    assertTrue(i.getTags().getItems().get(0).equals(NETWORK_TAGS.split(" ")[0]));
+    assertTrue(i.getTags().getItems().get(1).equals(NETWORK_TAGS.split(" ")[1]));
 
     // IAM
-    assert (i.getServiceAccounts().get(0).getEmail().equals(SERVICE_ACCOUNT_EMAIL));
+    assertTrue(i.getServiceAccounts().get(0).getEmail().equals(SERVICE_ACCOUNT_EMAIL));
 
     // Disks
-    assert (i.getDisks().get(0).getAutoDelete().equals(BOOT_DISK_AUTODELETE));
-    assert (i.getDisks().get(0).getBoot().equals(true));
-    assert (i.getDisks().get(0).getInitializeParams().getDiskType().equals(BOOT_DISK_TYPE));
-    assert (i.getDisks()
-        .get(0)
-        .getInitializeParams()
-        .getDiskSizeGb()
-        .equals(Long.parseLong(BOOT_DISK_SIZE_GB_STR)));
-    assert (i.getDisks()
-        .get(0)
-        .getInitializeParams()
-        .getSourceImage()
-        .equals(BOOT_DISK_IMAGE_NAME));
+    assertTrue(i.getDisks().get(0).getAutoDelete().equals(BOOT_DISK_AUTODELETE));
+    assertTrue(i.getDisks().get(0).getBoot().equals(true));
+    assertTrue(i.getDisks().get(0).getInitializeParams().getDiskType().equals(BOOT_DISK_TYPE));
+    assertTrue(
+        i.getDisks()
+            .get(0)
+            .getInitializeParams()
+            .getDiskSizeGb()
+            .equals(Long.parseLong(BOOT_DISK_SIZE_GB_STR)));
+    assertTrue(
+        i.getDisks().get(0).getInitializeParams().getSourceImage().equals(BOOT_DISK_IMAGE_NAME));
 
     InstanceConfiguration instanceConfiguration = instanceConfiguration();
-    assert (!instanceConfiguration.isUseInternalAddress());
-    assert (instanceConfiguration.instance().getMinCpuPlatform() == null);
+    assertTrue(!instanceConfiguration.isUseInternalAddress());
```

Comment:
`assertTrue(!` -> `assertFalse(`

Reply:
You caught me doing a string-replace-all ;)

Reply:
Done.